### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by bob-jackson**
Bi-weekly release notes.

### CODE REVIEW

File: Release Notes/2017/2017-05-23-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Load Balancer as a Service has added URI as a supported load balancing method. This new capability is now available in addition to Least Connection, Round Robin, and Source IP support.
+
+* __Managed Applications__
+
+Application Product Engineering has tested and certified Apache Tomcat
+7.0.77, 8.0.43 and 8.5.14 on RHEL  6 and 7.
+
+The Software Policy to use for RHEL 6 and 7 have been combined into a single policy.  The script which installs the package is now able to detect which major version of RHEL is targeted for the install, and utilizes the appropriate RHN channel for the installation.
+
+Installation Instructions: https://confluence.savvis.net/pages/viewpage.action?pageId=114091033
```
Comment (chrislittle) [inline Release Notes/2017/2017-05-23-cloud-platform-release-notes.md]: this is not a valid path to a public location for customers.  please adjust to an actual KB article to which this applies.  links to private CTL assets is not for public kbs

File: Release Notes/2017/2017-05-23-cloud-platform-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### New Features (1)
+
+* __Cloud Application Manager__
+
+- Bulk Import of auto discovered instances
+Cloud Application Manager now allows users to register auto discovered instances in bulk. Full instructions on how to use this new feature are available at the Knowledge Base section on our website.
+
+- Microsoft Azure Resale available for U.K. based customers
+Azure resale to companies based in the United Kingdom is Live! From the Azure Provider, users whose main registered address with CenturyLink is in the United Kingdom, will be permitted to automatically provision new Azure accounts for consolidated billing and platform level support. The capabilities that CenturyLink will Resell will still be the same as defined here https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-azure-capabilities/
```
Comment (chrislittle) [inline Release Notes/2017/2017-05-23-cloud-platform-release-notes.md]: should be linking to the kb directly in code/markdown

**Comment by bob-jackson** [comment]
Not sure what this means. The link works fine for me.

bob

Bob Jackson
Sr. Lead Product Owner
CenturyLink
A: 575 Maryville Centre, Town & Country, MO  63141
M: 636.395.0151 | E: bob.jackson@ctl.io<mailto:bob.jackson@ctl.io>

**Comment by chrislittle** [comment]
@bob-jackson per normal kb code if you are linking to another kb article you must reference it as code.  see https://github.com/CenturyLinkCloud/PublicKB#links 

for example:
The capabilities that CenturyLink will Resell will still be the same as defined [here](../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-azure-capabilities.md)

actual code:
```
The capabilities that CenturyLink will Resell will still be the same as defined [here.](../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-azure-capabilities.md)
```

File: Release Notes/2017/2017-05-23-cloud-platform-release-notes.md (hunk 2 above)
Comment (srveit) [inline Release Notes/2017/2017-05-23-cloud-platform-release-notes.md]: Change link to `../../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-azure-capabilities.md`

**Comment by bob-jackson** [comment]
Thx Stephan. I was just taking a look at needing an extra ../ when I got this. Hopefully, it is correct now.

Bob Jackson
Sr. Lead Product Owner
CenturyLink
A: 575 Maryville Centre, Town & Country, MO  63141
M: 636.395.0151 | E: bob.jackson@ctl.io<mailto:bob.jackson@ctl.io>

**Comment by bob-jackson** [comment]
Thx.

Bob Jackson
Sr. Lead Product Owner
CenturyLink
A: 575 Maryville Centre, Town & Country, MO  63141
M: 636.395.0151 | E: bob.jackson@ctl.io<mailto:bob.jackson@ctl.io>

**Pull request merged**